services/QuillLMS/app/models/user.rb

```diff
@@ -616,6 +603,10 @@ def completed_activity_count
     user_activity_classifications.sum(:count)
   end
 
+  def subscription_status
+    subscription&.subscription_status || last_expired_subscription&.subscription_status
```

Comment:
I might be missing something, but is it a problem that the old version of code does return an object if the user has no subscription, and here we'd always be returning nil?